pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -107,13 +109,13 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       std::istringstream f(custom_ops);
       std::string s;
       while (getline(f, s, ';')) {
-          white_list.insert(s);
+          white_list2.insert(s);
```

Comment:
any tools like `absl::StrSplit` in pytorch? better to not do it manually ...

Reply:
done.